src/components/SocialShare/SocialShare.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <div class="social-share">
+    <button class="button button--round" @click="toggleOptions">
+      <SvgIcon name="share-icon" class="button--round__icon" />
+      <span class="a11y-sr-only">
+        <slot>{{
+          !optionsAreVisible
```

Comment:
If a condition has both an if and an else, always use positive expression (don't negate):

```ts
optionsAreVisible
  ? $t("hideSharingOptions")
  : $t("showSharingOptions")
```